WebRoot/messages/ZaMsg_iw.properties

```diff
@@ -2304,6 +2304,8 @@ ss_inactive_accounts_30 = \u05d7\u05e9\u05d1\u05d5\u05e0\u05d5\u05ea \u05dc\u05d
 ss_inactive_accounts_90 = \u05d7\u05e9\u05d1\u05d5\u05e0\u05d5\u05ea \u05dc\u05d0-\u05e4\u05e2\u05d9\u05dc\u05d9\u05dd (90 \u05d9\u05d5\u05dd)
 
 #Busy Messages
+BUSY_LOADING = \u05d8\u05d5\u05e2\u05df\u002e\u002e\u002e
```

Comment:
Hi @k-kato , this file was missed, I have added the translations now.Can you review them?

Reply:
thank you for the update.